### PR TITLE
V1.3 tagdelete

### LINKED ISSUE #47: As an user, I can delete a tag
so that I can remove tags that are accidentally added.

### PR DESCRIPTION
**Pull request by JoeyToh**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/tagcommands/TagDeleteCommand.java
Comment (yeo-kexin) [inline src/main/java/seedu/address/logic/commands/tagcommands/TagDeleteCommand.java]: ah i think it is possible for there to be no contacts with a tag to delete since `tag add` creates a tag with no person attached to it? so maybe we shouldn't assert it haha but I'm not really sure!

**Review by yeo-kexin — COMMENTED** [review]
if it's not possible for tag to have no one attached to it, then LGTM!

File: src/main/java/seedu/address/logic/commands/tagcommands/TagDeleteCommand.java
Comment (siqiang-ng) [reply src/main/java/seedu/address/logic/commands/tagcommands/TagDeleteCommand.java]: I think so too!

File: src/main/java/seedu/address/logic/commands/TagDeleteCommand.java
Comment (siqiang-ng) [inline src/main/java/seedu/address/logic/commands/TagDeleteCommand.java]: Can I check why are there two TagDeleteCommand.java?

**Review by siqiang-ng — COMMENTED** [review]
Overall LGTM except to answer the few questions above^ before merging?

File: src/main/java/seedu/address/logic/commands/tagcommands/TagDeleteCommand.java
Comment (JoeyToh) [reply src/main/java/seedu/address/logic/commands/tagcommands/TagDeleteCommand.java]: ok I removed the assertion!

File: src/main/java/seedu/address/logic/commands/TagDeleteCommand.java
Comment (JoeyToh) [reply src/main/java/seedu/address/logic/commands/TagDeleteCommand.java]: Oops sorry my bad, deleted one of them.

**Review by siqiang-ng — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/model/tag/TagName.java
Comment (khinkhinn) [inline src/main/java/seedu/address/model/tag/TagName.java]: maybe can just call .tagName directly?

**Review by khinkhinn — COMMENTED** [review]
Minor comment, sorry to nitpick

**Pull request merged**
